simple/src/MediumProblem/TaxCalculator.java

```diff
@@ -0,0 +1,44 @@
+package MediumProblem;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TaxCalculator {
+    static float tax;
+
+    public static float IncomeTax(float income) {
+        if (income <= 500000) {
+            System.out.println("You are Tax free");
+        } else if (income >= 500000 && income < 700000) {
+            float taxableAmountOne = income - 500000;
+            tax = (0.10f) * taxableAmountOne;
+            System.out.println("Your are tax is :" + tax);
```

Comment:
You can create one common variable to store taxableAmount, anyway only 1 loop is going to be executed